include/madrona/mw_gpu.hpp

```diff
@@ -45,6 +45,16 @@ struct StateConfig {
 
     // Number of exported ECS components
     uint32_t numExportedBuffers;
+
+    // This is required if the BVH is to be invoked (holds object information).
+    imp::ImportedAssets::GPUGeometryData *geometryData;
+
+    Span<const imp::SourceMaterial> materials;
+    Span<const imp::SourceTexture> textures;
+
+    uint32_t raycastOutputResolution;
+
+    float nearSphere;
```

Comment:
What is this?

Reply:
again, not supposed to be there - this config is now moved to RenderConfig (it's now nearPlane - I called it near sphere because it's technically a sphere with the tracing).